fontir/src/ir.rs

```diff
@@ -81,6 +82,18 @@ pub struct MiscMetadata {
 
     pub underline_thickness: OrderedFloat<f32>,
     pub underline_position: OrderedFloat<f32>,
+
+    /// UFO appears to allow negative major versions.
```

Comment:
hm I believe that's just a typo.. It doesn't make any sense to have version -1.0... 🤯

Reply:
norad concurs so while I agree -1.0 is a very very weird version it does appear possible.

Reply:
feel free to bump https://github.com/unified-font-object/ufo-spec/issues/215